pyproject.toml

```
@@ -1,3 +1,35 @@
+[project]
+name = "s2geometry"
+description = "Computational geometry and spatial indexing on the sphere"
+authors = [
+    { name = "Dan Larkin-York", email = "dan@arangodb.com" },
+    { name = "Google Inc." },
+    { name = "Koordinates Limited" },
+    { name = "Mike Playle", email = "mike@mythik.co.uk" },
+    { name = "Tiago Brito", email = "tiago.brito@90poe.io" },
+]
+requires-python = ">=3.7"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Operating System :: POSIX",
+    "License :: OSI Approved :: Apache Software License",
+]
+dynamic = [
+    "version",
+]
+
+[project.license]
+file = "LICENSE"
```

There's no way to use an SPDX identifier here?

---

sure, it can be 
```suggestion
text = "Apache 2"
```
but then, why have a license file? I personally prefer reducing the amount of duplication in configuration but we can definitely change it to be the above if you prefer